### ISSUE TITLE
Decide what to do with metal_BERT_large_11 because it hangs on WH in CI

### ISSUE THREAD

**Issue by tt-rkim** [tenstorrent/tt-metal#7525]
Skipping in CI.

https://github.com/tenstorrent/tt-metal/actions/runs/8711779487/job/23896836983

cc: @TT-billteng @jliangTT @tarafdarTT @TT-BrianLiu @davorchap

**Comment by tt-rkim** [comment]
@jliangTT @boris-drazic @uaydonat

This is an older implementation of BERT, before Javelin.

Should we keep this around?

**Comment by tt-rkim** [comment]
metal_BERT_large_11 fails with DRAM with sharded config in Wormhole nightly CI

but it hangs on `test_demo.py` on wormhole

**Comment by tt-aho** [comment]
This is not an old implementation, and it should be kept around. This is the performant WH implementation of bert in tt_lib

**Comment by tt-rkim** [comment]
This is relevant again.

Hangs on WH device perf

**Comment by tt-aho** [comment]
This is due to suspected di/dt right? I think @rtawfik01 said we can remove it from ci while it's being looked into.

**Comment by rtawfik01** [comment]
Yes we can skip these tests until we can reproduce locally and make sure its di/dt or if its a potential other issue
